Directory.Build.props

```diff
@@ -21,4 +21,22 @@
 
   </PropertyGroup>
 
+  <PropertyGroup Label="Analysis rules">
+
+    <AnalysisLevel>latest-Recommended</AnalysisLevel>
+
+    <!-- Prefix generic type parameter name with 'T'  -->
+    <NoWarn>$(NoWarn);CA1715;</NoWarn>
+
+    <!-- Do not declare visible instance fields  -->
+    <NoWarn>$(NoWarn);CA1051;</NoWarn>
```

Comment:
If this is for the CompilationHelper then maybe I should just use properties or readonly fields.

Reply:
It's also all structs which don't use properties but public fields.